### ISSUE TITLE
Agree on Meeting-Structure

### ISSUE THREAD

**Issue by diva-exchange** [Shoalsteed/Community-Meeting#2]
Structure means:
* What is the purpose of the meeting, who shall be there?
* Date/Time, like once a month, should probably be good for Northern America and Europe?
* Length, like 15 minutes or 30 minutes?
* How is moderation organized? Talk time limitations, like maximum 120 secs per participant over all.
* What is the technical platform? Video/Audio only? Exclude anonymous participants and text-only (irc)?
* How to manage topics? Github issues?
* How to organize notes-taking (like round-robin)?

**Comment by Shoalsteed** [comment]
- introduce new apps, research, use cases.
- Once a month.
- Start with a half hour. 
- Moderation: I think that ideally we would have people submit a light proposal, ie " I would like to present an opportunity to review and discuss my research, or demonstrate a new app, or perhaps a fun activity to create personas or use cases. 
This would help for keeping time/ talk time. I need to explore moderation settings for platforms. We will need to agree on participation standards. 
- I would prefer to use video and audio. Jitsi does have an option to not use a mic or video, so there can be chat participation. 
- Yes, manage topics in git repos seems okay for now. Open to other ideas. 
- Good question: I think we agree on who does the note taking. I also would like to be publishing outcome to The Overlay blog.

**Comment by diva-exchange** [comment]
Proposal to vote for:

* Jitsi meeting (meet.jit.si) - using this URL scheme: meet.jit.si/I2P{YYYY-MM-DD} - like meet.jit.si/I2P2023-01-18
* Language: English
* Date suggestion: every third Wednesday each month
* Time: 19:30 - 19:30 CET, 1pm - 1:30pm EST
* Participants may propose 5 min talks until second Wednesday each month, using this repo (by adding the proposal to a given issue in this repo). 5 min talk + 5 min discussion = max. three talks each meeting.
* Moderators: round-robin selection, starting with @Shoalsteed ; next moderator choosen at the end of the meeting
* Moderator takes a final decision on proposed talks, if there are more than three proposals - decision deadline: second Thursday each month (just after the proposal deadline)
* Simple moderation: just watch talk/discussion times and make sure everyone can speak up
* Moderator = note taker; Notes = max. 2 sentences per 5 minutes - added to the related notes-issue in this repo - those which have a talk can add more notes or files to the note themselves, if they feel it's needed
* First meeting 18-Jan-2023, deadline for proposal 11-Jan-2023
* Code of Conduct (standard stuff): https://github.com/diva-exchange/divachain/blob/develop/CODE_OF_CONDUCT.md

@Shoalsteed Could you please motivate the I2P community to look at this comment and to vote for it using thumb up or thumb down?

**Comment by Shoalsteed** [comment]
I propose a more simplified Code of Conduct that I will create for review. 

We also need to provide specific areas of interest to help contributors. I want people who work in localization and digital security trainers and designers, other privacy projects to feel confident to attend and participate. 

@diva-exchange When posting to Twitter, please use the project handle, otherwise I do not see the post!
@GetI2P (Twitter) 
@i2p@mastodon.social ( Mastodon)

**Comment by eyedeekay** [comment]
This would be distinct from the existing development meeting which still takes place on IRC2P, correct? I'm leading that tomorrow and have been leading that monthly, normally on the first Tuesday when it doesn't get rescheduled.

**Comment by Shoalsteed** [comment]
@eyedeekay Yes - this is distinct. It is not related to the Java dev meeting. This is meant to focus on showcasing an app, or research, or things that may have a demo element. It will be more inline with DWeb style meetings to hopefully encourage more people to participate in testing, building and getting to know the I2P network and its capacity. A great opportunity especially for showcasing new Windows installer or Firefox extension!

**Comment by diva-exchange** [comment]
@Shoalsteed @eyedeekay 
1. Thanks for the social media handles :) - will make sure, that this gets respected
2. Could a **reddit mod please pin** this issue on /r/i2p to get more reach and feedback?
3. **Decision** could be taken regarding first meeting in early January 2023
4. BTW: shortest possible Code-of-Conduct: "Be nice. Be on topic. Be constructive. Only 100% open source technology and publicly available research."

**Comment by Shoalsteed** [comment]
@diva-exchange I will create a post early next week for Reddit. I am away until then due to a family emergency.

**Comment by Shoalsteed** [comment]
@diva-exchange Please evaluate for announcement.

Event: January 18th, 2023
Time: 19:30 - 19:30 CET, 1pm - 1:30pm EST
Deadline for presenting : January 11th, 2023
Recording of the event: TBD/ Optional
Platform: Jitsi

**House Rules For All Participants**
Only 100% Open Source work and research. 
Be courteous.
Be considerate.
Be constructive. 
This is not an opportunity to create division between distributions or other privacy tools.


**How to submit a talk**
Create a new issue in this repo. 
It should include a link to your Github/lab repo if you are presenting work or research. It should let us know if you will be using screen sharing. Please provide a brief description of your proposed talk, and any privacy preferences since we will be providing blog posts and sharing to social media.

**Comment by diva-exchange** [comment]
@Shoalsteed Great! One remark: "Create a new issue in this repo: [add link to repo here]"

Post & Pin it to reddit, please. As soon as it's on reddit: I will initiate posting on masto, twitter and create a blog post about it in +/- early January 2023.

**Comment by Shoalsteed** [comment]
I will post to Reddit on Monday - I will not be available to manage any questions since I am taking the weekend to be with my family. I have updated the readme.md. !
